include/interface/blas2_interface.h

```diff
@@ -515,6 +541,36 @@ typename sb_handle_t::event_t inline _syr(
   return internal::_syr(sb_handle, _Uplo, _N, _alpha, _vx, _incx, _mA, _lda);
 }
 
+/*!
+ @brief Generalised vector squaring followed by a sum with a packed symmetric
+ matrix.
+
+ Generalised vector squaring followed by a sum with a packed symmetric matrix,
+ i.e. computing the mathematical operation:
+
+ A = alpha*x*xT + A
+
+ See the netlib blas interface documentation for more details of the high level
+ interface:
+ https://netlib.org/lapack/explore-html/d6/d30/group__single__blas__level2_ga13dca9a765471c68248e8e00190f4d4d.html#ga13dca9a765471c68248e8e00190f4d4d
```

Comment:
```suggestion
https://netlib.org/lapack/explore-html/dd/dba/dspr_8f.html
```
For consistency with the other functions.